docs/source/guides/rpakmodding.rst

```diff
@@ -0,0 +1,236 @@
+    for example ``256x256`` ``512x512`` ``1024x512`` ``4096x1024`` are all fine, but ``350x700`` might cause issues.
+
+Place your newly created .dds file in the ``assets\texture`` folder, following the path in the Name you noted down above.
+In this example the .dds file would go in ``RePak\assets\texture\models\camo_skins``, with the path of the image being ``..\RePak\assets\texture\models\camo_skins\camo_skin04_col.dds``
+
+
+Making a map file
+-----------------
+
+Once you have edited your texture image and placed it in the right folder, you are ready to make your map file.
+
+Map files are what RePak uses to create the .rpak file (and .starpak files if needed) and are in the .json file format.
+They can be named anything you want, but should be put in the ``RePak\maps`` folder.
+
+Below is an example of a map file that creates an RPak called ``example.rpak`` which contains 1 texture asset.
+
+.. code-block:: json
+
+    {
+        "name":"example",
+        "assetsDir":"../assets",
+        "outputDir":"../rpaks",
+        "version": 7,
+        "files":[
+            {
+                "$type":"txtr",
+                "path":"texture/models/camo_skins/camo_skin04_col"
+            }
+        ]
+    }
+
+- ``name``: the name of the file that gets created by RePak.
+- ``assetsDir``: the folder that RePak bases the file path on when looking for textures.
+- ``outputDir``: the folder that RePak will put the files that it creates in.
+- ``version``: the RPak version RePak will use when creating the RPaks. **Version 7 is Titanfall 2, version 8 is Apex Legends.**
+- ``files``: an array of all of the assets that RePak will create in the RPak.
+- ``$type``: the type of asset that this asset is, use ``txtr`` for textures.
+- ``path``: the path of the asset, used in texture assets for finding the image. **This should start with** ``texture/`` **and the rest should match the Name given by LegionPlus.**
+
+.. warning:: 
+    If the ``path`` doesn't match up with the location of your file, RePak will throw an error
+
+.. warning::
+    If the ``path`` contains any ``\`` characters, make sure that you either replace them with ``/`` or you duplicate them (``\\``)
+
+    This is because ``\`` is the escape character in JSON, and will therefore break the ``path``
+
+Creating the RPak
+-----------------
+
+To create your RPak file, simply open ``pack_all.bat``.
+
+Alternatively, click and drag your map file over ``RePak.exe``. (I don't recommend this, it's a pain)
+
+**Look at the console for any errors.**
+If there are no errors, a .rpak file should have been created in the ``rpaks`` folder.
+
+
+Using the RPak in a mod
+-----------------------
+
+Create the basis of the mod using the :doc:`gettingstarted` guide.
+
+Inside the mod's folder, create a new folder, called ``paks``. Move your .rpak file (and .starpak files if you have any) into the folder.
+
+|ModStructure|
+
+Inside the ``paks`` folder that you created, make a new .json file called ``rpak``.
```

Comment:
```suggestion
Inside the ``paks`` folder that you created, make a new .json file called ``rpak.json``.
```
Dont know if this is what is meant here, but it if is, this phrasing makes it more clear what you should call the file.